app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
```

Comment:
You don`t have to import these modules, your code will work without them.

Reply:
I have becouse class Dwarf  and class Elf were appied in  def elves_concert, def feast_of_the_dwarves